src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/ApplicationFormController.java

```
@@ -264,6 +270,26 @@ public ResponseEntity<String> getApplicationStatus(@PathVariable final Integer a
         return ResponseEntity.ok(applicationStatus.toString());
     }
 
+    @GetMapping("/{applicationId}/download-summary")
+    @CheckSchemeOwnership
+    public ResponseEntity<ByteArrayResource> exportApplication(
+            @PathVariable final Integer applicationId, HttpServletRequest request) {
+        try (OdfTextDocument odt = applicationFormService.getApplicationFormExport(applicationId)) {
+
+            ByteArrayResource odtResource = odtService.odtToResource(odt);
+
+            HttpHeaders headers = new HttpHeaders();
+            headers.add(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=\"application.odt\"");
+            headers.add(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_OCTET_STREAM_VALUE);
+
+            return ResponseEntity.ok().headers(headers).contentLength(odtResource.contentLength())
+                    .contentType(MediaType.APPLICATION_OCTET_STREAM).body(odtResource);
+        } catch (Exception e) {
+            log.error("Could not generate ODT. Exception: ", e);
+            throw new RuntimeException(e);
```

Can we throw a custom exception here with a specific status code?